.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
-          lfs: true
+          lfs: false
```

Comment:
`false` is the default but I'm leaving it explicitly disabled to avoid someone unintentionally re-enabling